lib/instrument/transformer/nskeyed.js

```diff
@@ -203,11 +205,87 @@ class XCTCapabilities extends BaseArchiveHandler {
 }
 
 class NSNull extends BaseArchiveHandler {
+  static isNSObject = true;
   decodeArchive () {
     return null;
   }
 }
 
+class XCTestConfiguration extends BaseArchiveHandler {
+  static _default = {
+    aggregateStatisticsBeforeCrash: {
+      XCSuiteRecordsKey: {}
+    },
+    automationFrameworkPath: '/Developer/Library/PrivateFrameworks/XCTAutomationSupport.framework',
+    baselineFileRelativePath: undefined,
+    baselineFileURL: undefined,
+    defaultTestExecutionTimeAllowance: undefined,
+    disablePerformanceMetrics: false,
+    emitOSLogs: false,
+    formatVersion: 2,
+    gatherLocalizableStringsData: false,
+    initializeForUITesting: true,
+    maximumTestExecutionTimeAllowance: undefined,
+    productModuleName: 'WebDriverAgentRunner',
+    randomExecutionOrderingSeed: undefined,
+    reportActivities: true,
+    reportResultsToIDE: true,
+    systemAttachmentLifetime: 2,
+    targetApplicationArguments: [],
+    targetApplicationBundleID: undefined,
+    targetApplicationEnvironment: undefined,
+    targetApplicationPath: '/KEEP-THIS-NOT-EMPTY/KEEP-THIS-NOT-EMPTY',
+    testApplicationDependencies: {},
+    testApplicationUserOverrides: undefined,
+    testBundleRelativePath: undefined,
+    testExecutionOrdering: 0,
+    testTimeoutsEnabled: false,
+    testsDrivenByIDE: false,
+    testsMustRunOnMainThread: true,
+    testsToRun: undefined,
+    testsToSkip: undefined,
+    treatMissingBaselinesAsFailures: false,
+    userAttachmentLifetime: 1
+  }
+  constructor(data) {
+    super();
+    if (!data.testBundleURL) {
```

Comment:
Should here allow to set `productModuleName` as another value to make this not for WDA specific?
(Just for the balance of the class's naming and `productModuleName`)

Reply:
name is actually not functional mattered. But I got your point, name could be a optional parameter in Xctest class